examples/cpuio.fio

```diff
@@ -1,8 +1,15 @@
 [global]
 ioengine=cpuio
 time_based
-runtime=10
+runtime=15
 
-[burn50percent]
+[burn_2x50_noop]
 cpuload=50
+numjobs=2
+cpumode=noop
 
+[burn_2x50%_qsort]
+stonewall
+cpuload=50
+numjobs=2
+cpumode=qsort
```

Comment:
Should probably include a comment here, it's nice if examples are well documented.

Reply:
sure.